packages/terra-graphs-docs/src/example-datasets/dataObjects/General/dataset2.js

```diff
@@ -0,0 +1,25 @@
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
```

Comment:
Instead of dataset2, can you name this something related to examples.( Probably do the same for other datasets).